src/gluonts/model/deepstate/issm.py

```diff
@@ -186,6 +192,9 @@ class LevelTrendISSM(LevelISSM):
     def latent_dim(self) -> int:
         return 2
 
+    def output_dim(self) -> int:
+        return 1
```

Comment:
We need to update ISSM to handle multi-variate case; this can be set from outside then.

(I will open a new pull request for this.)